reporter/src/funTest/assets/static-html-reporter-test-expected-output.html

```diff
@@ -562,7 +562,8 @@ <h3 class="">Packages</h3>
             <td><a href="#Gradle:com.here.ort.gradle.example:lib:1.0.0-pkg-1">1</a></td><td>Gradle:com.here.ort.gradle.example:lib:1.0.0</td><td></td><td><em>Detected Licenses:</em>
               <dl>
                 <dd>
-                  <div>MIT</div>
+                  <div>Apache-2.0 (multi-line) (<a href="https://github.com/heremaps/oss-review-toolkit/blob/master/analyzer/src/funTest/assets/projects/synthetic/gradle/lib/build.gradle#L19-L20">link</a> to the location)</div>
+                  <div>Apache-2.0 (single-line) (exemplary <a href="https://github.com/heremaps/oss-review-toolkit/blob/master/analyzer/src/funTest/assets/projects/synthetic/gradle/lib/build.gradle#L19">link</a> to the first of 2 locations)</div>
```

Comment:
The entries now take multiple lines, can you add the following to `static-html-reporter.css` above the entry for `<ul>` to make it look a bit nicer? This reduces the total indentation of the entries and adds indentation from the second line of each entry onwards.

```css
dd {
    margin-left: 1em;
}

dd div {
    padding-left: 1em;
    text-indent:-1em;
}
```

Reply:
Done, thanks for the suggestion.